doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -275,6 +299,204 @@ The :c:func:`lte_lc_env_eval` function starts the environment evaluation for the
+==============
+
+The LTE link control library provides functionality to manage Packet Data Protocol (PDP) contexts and Packet Data Network (PDN) connections.
+
+The PDN functionality provides the following capabilities:
+
+* Creating and configuring PDP contexts
+* Receiving events pertaining to PDN connections
+* Managing PDN connections
+* Retrieving PDN connection information
+
+To enable PDN functionality, set the :kconfig:option:`CONFIG_LTE_LC_PDN_MODULE` Kconfig option to ``y``.
+
+AT commands used
+----------------
+
+The PDN functionality uses several AT commands, and it relies on the following two types of AT notifications to work:
+
+* Packet domain events notifications (``+CGEV``) - Subscribed by using the ``AT+CGEREP=1`` command.
+  See the `AT+CGEREP set command`_ section in the nRF9160 AT Commands Reference Guide or the `nRF91x1 AT+CGEREP set command`_  section in the nRF91x1 AT Commands Reference Guide, depending on the SiP you are using.
+* Notifications for unsolicited reporting of error codes sent by the network (``+CNEC``) - Subscribed by using the ``AT+CNEC=16`` command.
+  See the `AT+CNEC set command`_ section in the nRF9160 AT Commands Reference Guide or the `nRF91x1 AT+CNEC set command`_  section in the nRF91x1 AT Commands Reference Guide, depending on the SiP you are using.
+
+The LTE link control library automatically subscribes to the necessary AT notifications.
+This includes automatically resubscribing to the notifications upon functional mode changes.
+
+.. note::
+   The subscription to AT notifications is lost upon changing the modem functional mode to ``+CFUN=0``.
+   If the application subscribes to these notifications manually, it must also take care of resubscription.
+
+Following are the AT commands that are used by the PDN functionality:
+
+* ``AT%XNEWCID`` - To create a PDP context
+* ``AT+CGDCONT`` - To configure or destroy a PDP context
+* ``AT+CGACT`` - To activate or deactivate a PDN connection
+* ``AT%XGETPDNID`` - To retrieve the PDN ID for a given PDP context
+* ``AT+CGAUTH`` - To set the PDN connection authentication parameters
+
+For more information about these commands, see `Packet Domain AT commands`_ in the nRF9160 AT Commands Reference Guide or the `nRF91x1 packet Domain AT commands`_ section in the nRF91x1 AT Commands Reference Guide, depending on the SiP you are using.
+
+Creating and managing PDN connections
+--------------------------------------
+
+The application can create PDP contexts by using the :c:func:`lte_lc_context_create` function.
+PDN events are sent to all registered LTE link control event handlers.
+The application can use the :c:func:`lte_lc_register_handler` function to register an event handler for events pertaining to all PDP contexts, including the default PDP context.
+
+The :c:func:`lte_lc_context_configure` function is used to configure a PDP context, which can be configured with address family, access point name, and optional additional configuration options.
```

Comment:
With an introductory sentence, you might list the functions below as bullet points.